integration/testnet/network.go

```diff
@@ -754,20 +796,20 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 
 			nodeContainer.bindPort(hostPort, containerPort)
 
-			hostMetricsPort := testingdock.RandomPort(t)
-			containerMetricsPort := "8080/tcp"
+			// hostMetricsPort := testingdock.RandomPort(t)
+			// containerMetricsPort := "8080/tcp"
 
-			nodeContainer.bindPort(hostMetricsPort, containerMetricsPort)
-			net.MetricsPortsByContainerName[nodeContainer.Name()] = hostMetricsPort
+			// nodeContainer.bindPort(hostMetricsPort, containerMetricsPort)
+			// net.MetricsPortsByContainerName[nodeContainer.Name()] = hostMetricsPort
 
 			nodeContainer.AddFlag("rpc-addr", fmt.Sprintf("%s:9000", nodeContainer.Name()))
 
 			nodeContainer.Ports[ExeNodeAPIPort] = hostPort
 			nodeContainer.opts.HealthCheck = testingdock.HealthCheckCustom(healthcheckExecutionGRPC(hostPort))
 			net.AccessPorts[ExeNodeAPIPort] = hostPort
 
-			nodeContainer.Ports[ExeNodeMetricsPort] = hostMetricsPort
-			net.AccessPorts[ExeNodeMetricsPort] = hostMetricsPort
+			// nodeContainer.Ports[ExeNodeMetricsPort] = hostMetricsPort
+			// net.AccessPorts[ExeNodeMetricsPort] = hostMetricsPort
```

Comment:
still needed?

Reply:
yeah, @kc1116 still need this for debugging purpose

Reply:
It is useful when you need to view metrics for a particular server. Can you refactor this into a separate func that can be used to expose metrics on a single container in the network? It is a nice to have.